ga4gh/datamodel/variants.py

```diff
@@ -1096,8 +1096,11 @@ def convertVariantAnnotation(self, record, transcriptConverter):
                 annotations, variant, hgvsG, transcriptConverter)
         else:
             annotations = record.info.get('CSQ'.encode())
-            transcriptEffects = self.convertTranscriptEffectCSQ(
-                annotations[0], hgvsG)
+            transcriptEffects = []
+            for ann in annotations:
+                transcriptEffects += \
```

Comment:
Yes, sorry, my bad. The advantage of using .extend rather than += here is that we can use an arbitrary iterable as the argument. += will choke if the other argument isn't a list.